### PR TITLE
fixed bugs and added more buttons

### PR DESCRIPTION
**Pull request by trains2050**
Hello!
I fixed some bugs and added more buttons to the mkpc presences! Also added support for the staff backend!
Proof: 
image

### CODE REVIEW

File: websites/M/MKPC/presence.ts [hunk 1]
```diff
@@ -41,16 +39,26 @@ presence.on("UpdateData", async () => {
...
+    presenceData.buttons = [{ label: "View topic", url: document.location.href }];
+  } else if (
+    document.location.pathname == "/ban-player.php" ||
+    document.location.pathname == "/admin.php" ||
+    document.location.pathname == "doublecomptes.php"
+  ) {
     presenceData.startTimestamp = browsingStamp;
-    user = document.querySelector("html > body > main > h1");
+    presenceData.details = "Viewing staff backend";
+  } else if (document.location.pathname == "/profil.php") {
```
Comment (callumok2004) [inline websites/M/MKPC/presence.ts:56]: shouldn't `profil` have an `e` at the end

**Comment by callumok2004** [comment]
Oh, and show the staff backend part of it working with a screenshot

File: websites/M/MKPC/presence.ts (hunk 1 above)
Comment (trains2050) [reply websites/M/MKPC/presence.ts:56]: i know, its the location of the website thats spelled like this xd, even i was confused but the location is /profil.php

**Comment by trains2050** [comment]
> Oh, and show the staff backend part of it working with a screenshot
here:
image

**Comment by trains2050** [comment]
> Oh, and show the staff backend part of it working with a screenshot >

here: https://github.com/PreMiD/Presences/pull/3543#issuecomment-821988373 
can not show more info due to privacy etc

File: websites/M/MKPC/presence.ts [hunk 2]
```diff
@@ -41,16 +39,26 @@ presence.on("UpdateData", async () => {
...
+  ) {
     presenceData.startTimestamp = browsingStamp;
-    user = document.querySelector("html > body > main > h1");
+    presenceData.details = "Viewing staff backend";
+  } else if (document.location.pathname == "/profil.php") { 
+    presenceData.startTimestamp = browsingStamp;
+    user = document.querySelector("body > main > div > div.profile-summary > h1");
     presenceData.details = "Viewing: " + user.innerHTML;
     presenceData.smallImageKey = "search";
+    presenceData.buttons = [{ label: "View user profile", url: document.location.href }];
```
Comment (EncryptedDev) [inline websites/M/MKPC/presence.ts:61]: View profile?

File: websites/M/MKPC/presence.ts [hunk 3]
```diff
@@ -20,18 +20,16 @@ presence.on("UpdateData", async () => {
...
     presenceData.startTimestamp = browsingStamp;
     presenceData.details = "Viewing the forum's menu";
-    presenceData.smallImageKey = "search";
-  } else if (document.location.pathname == "index.php") {
-    presenceData.startTimestamp = browsingStamp;
-    presenceData.details = "Viewing home page";
-  }
-  const elt = document.querySelector("#compteur0 > div") as HTMLElement;
+    presenceData.smallImageKey = "search"; 
+    presenceData.buttons = [{ label: "View forum menu", url: document.location.href }];
```
Comment (EncryptedDev) [inline websites/M/MKPC/presence.ts:24]: View Forum?

File: websites/M/MKPC/presence.ts [hunk 4]
```diff
@@ -20,18 +20,16 @@ presence.on("UpdateData", async () => {
...
+  } const elt = document.querySelector("#compteur0 > div") as HTMLElement;
   if (elt) {
     const lap = elt.innerText.replace(/.+? /g, "");
     presenceData.details = "On: Lap " + lap;
```
Comment (EncryptedDev) [inline websites/M/MKPC/presence.ts:28]: Lap <lap>?

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Look at my review, the wording of things seems off. Also, I think your indentation is messed

**Comment by EncryptedDev** [comment]
show proof of working

**Comment by trains2050** [comment]
> show proof of working

wording would not change anything code related but here is proof of working  on the changes i made and more proof can be seen above xd
image
image
image

**Comment by EncryptedDev** [comment]
you messed the branch lol

**Comment by trains2050** [comment]
> you messed the branch lol

how? i reverted the changes that messed it up lol, how should i fix it?

**Comment by callumok2004** [comment]
now you removed every change you made, can we get a clap clap

**Comment by ririxidev** [comment]
I will ask you one last time @callumok2004 @EncryptedDev are you in reviewer team? Nah. So stop. We know what we need to do and we are here to do it. Wanna be reviewers? Feel free ask Bas, and stop acting like you are reviewer.

**Comment by StrikerFRFX** [comment]
> I will ask you one last time @callumok2004 @EncryptedDev are you in reviewer team? Nah. So stop. We know what we need to do and we are here to do it. Wanna be reviewers? Feel free ask Bas, and stop acting like you are reviewer.

Maja, please take this business you have against other people reviewing to DMs privately. You're making a scene, and consult Bas about it because I know damn well you haven't spoken to him about it last I heard. It's a public Github repository for a fucking reason, and people can review and assist however they feel fit (and neither Callum nor Encrypted are reviewers but they actually point out things we as reviewers could simply miss). I don't seek to argue about it, but this is CONSTANT so sort it out privately.

**Comment by ririxidev** [comment]
Oh cool then! Find another reviewer! Thanks for everything!

**Pull request closed without merging**